src/Application/FileProcessor.php

```diff
@@ -142,10 +149,16 @@ private function printFile(File $file, Configuration $configuration, string $fil
              * Handle new line or space before <?php or InlineHTML node wiped on print format preserving
              * On very first content level
              */
-            $originalFileContent = $file->getOriginalFileContent();
-            $ltrimOriginalFileContent = ltrim($originalFileContent);
+            $ltrimOriginalFileContent = ltrim($file->getOriginalFileContent());
+            $ltrimNewContent = ltrim($newContent);
```

Comment:
no need ltrim on `$newContent` as before, php-parser v4 already ltrim it, on php-parser v5, the ltrim will no longer needed.